npm-install/action.yaml

```diff
@@ -0,0 +1,19 @@
+name: Set up deps
+description: Loads NPM dependencies for a CI job, installing them if not cached
+runs:
+  using: composite
+  steps:
+    - name: Set up Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '14'
```

Comment:
This should be a customizable input with no default value, to encourage every project to declare their desired Node.js version explicitly.